tests/dec/test_known_decay_models.py

```diff
@@ -15,27 +15,6 @@
 DIR = Path(__file__).parent.resolve()
 
 
-def test_lark_file_model_list_consistency():
```

Comment:
I've removed this test since it doesn't really make sense anymore. I'm not sure how to replace it with something new though, any idea?

Reply:
The removal is logical given your move since there is no consistency to be checked between 2 "sources".